ui/src/literature/components/ImprintInfo.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { List } from 'immutable';
+
+import InlineList, { SEPARATOR_AND } from '../../common/components/InlineList';
+
+class ImprintInfo extends Component {
+  static renderImprints(imprints) {
```

Comment:
I would change the name of this function to `renderImprint` and an argument to a single `imprint`